src/aioswitcher/device/__init__.py

```diff
@@ -30,6 +30,7 @@ class DeviceCategory(Enum):
     POWER_PLUG = auto()
     THERMOSTAT = auto()
     SHUTTER = auto()
+    SHUTTER_SINGLE_LIGHT_DUAL = auto()
```

Comment:
I think it will make more sense to call it 
```suggestion
    SINGLE_SHUTTER_DUAL_LIGHT = auto()
```